luxai_s2/luxai_runner/process.py

```diff
@@ -54,7 +54,7 @@ async def start(self):
         base_file_path = os.path.basename(self.file_path)
         if self.is_binary:
             self._agent_process = await asyncio.create_subprocess_exec(
-                f"./{base_file_path}",
+                f"{cwd}\{base_file_path}" if "win" in sys.platform else f"./{base_file_path}",
```

Comment:
im surprised we need to provide the current working directory here. is this not handled by the cwd parameter in create_subprocess_exec?

Reply:
Yeah. For whatever reason windows utterly ignores the provided cwd and uses the cwd of the luxai-s2 process.